examples/morpheusvm/tests/workload/genesis.go

```diff
@@ -60,3 +69,39 @@ func NewDefaultKeys() []ed25519.PrivateKey {
 
 	return testKeys
 }
+
+type NetworkConfiguration struct {
```

Comment:
nit: we should add a type check for NetworkConfiguration

`var _ TestNetworkConfiguration = &NetworkConfiguration{}`

Reply:
done